### DISCUSSION TITLE
Connect tasks across DAGs (cross job dependency)

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#8484] [Q&A]
Similar questions have been asked several times in Dagster Slack:

I just started using Dagster, and am trying to grep the core ideas against airflow as my backdrop. One question I have is around the concept of multiple DAGs in Airflow and being able to connect tasks across DAGs. Is there a similar concept in Dagster, where I can define multiple jobs and have ops in a job depend on an op in another?

*[Original Slack thread](https://dagster.slack.com/archives/C01U5LFUZJS/p1638731624089300?thread_ts=1638731624.089300&cid=C01U5LFUZJS)*

There are two independent tasks "A" and "B".
Task "A" downloads data and starts every day at 05:00 AM.
Task B runs once a week. I want to launch task "B" once a week (on Monday), but after the completion of task "A".
How can you link independent tasks?

*[Original Slack thread](https://dagster.slack.com/archives/C01U954MEER/p1637732646054800)*

**Comment by yuhan** [comment]
[Asset sensors](https://docs.dagster.io/concepts/partitions-schedules-sensors/asset-sensors) is in particular one way to set up these types of dependencies. 

For example, job "A" runs, and has a terminal op which materializes an asset “task_A_completion_mark” (i.e. `yield AssetMaterialization(asset_key="task_A_completion_mark")` in the last op in job "A"). Job "B" is hooked up to an asset sensor listening for “task_A_completion_mark”, which potentially could run on every successful run of A, but decides to return a RunRequest if and only if the current day is Monday.
